sts_auth/okta.py

```diff
@@ -93,79 +90,76 @@ def get_verification_status_from_response(self, response):
+        adapter_glue_form = response.soup.find(id="adapterGlue")
         referer = response.url
-        self.session.headers.update({'Referer': referer})
+        self.session.headers.update({"Referer": referer})
         selectors = ",".join("{}[name]".format(i) for i in ("input", "button", "textarea", "select"))
-        data = [(tag.get('name'), tag.get('value')) for tag in adapter_glue_form.select(selectors)]
-        logger.debug('Posting data to url: {}\n{}'.format(referer, data))
+        data = [(tag.get("name"), tag.get("value")) for tag in adapter_glue_form.select(selectors)]
+        logger.debug("Posting data to url: {}\n{}".format(referer, data))
         return self.session.post(referer, data=data)
 
     def okta_totp_verification(self, factor_details):
         if not self.okta_shared_secret:
             logger.debug(
-                'TOTP Verification available but Okta Shared Secret '
-                'is not set. For instructions to set the Shared Secret, '
-                'refer to the README: '
-                'https://github.com/cshamrick/stsauth/blob/master/README.md'
+                "TOTP Verification available but Okta Shared Secret "
+                "is not set. For instructions to set the Shared Secret, "
+                "refer to the README: "
+                "https://github.com/cshamrick/stsauth/blob/master/README.md"
             )
             return False
 
         totp = pyotp.TOTP(self.okta_shared_secret)
-        verify_url = factor_details.get('_links', {}).get('verify', {}).get('href')
+        verify_url = factor_details.get("_links", {}).get("verify", {}).get("href")
         try:
             pass_code = totp.now()
         except Exception as e:
-            msg = 'An error occured fetching your TOTP code. Please check your Shared Secret.'
-            click.secho(msg, fg='red')
-            click.secho('Error: {}'.format(str(e)), fg='red')
+            msg = "An error occured fetching your TOTP code. Please check your Shared Secret."
+            click.secho(msg, fg="red")
+            click.secho("Error: {}".format(str(e)), fg="red")
             sys.exit(1)
         if verify_url:
             verify_response = self.session.post(
-                verify_url,
-                json={
-                    'stateToken': self.state_token,
-                    'passCode': pass_code
-                })
+                verify_url, json={"stateToken": self.state_token, "passCode": pass_code}
+            )
             if verify_response.ok:
-                status = verify_response.json().get('status')
-                if status == 'SUCCESS':
+                status = verify_response.json().get("status")
+                if status == "SUCCESS":
                     return True
         click.secho(
-            'TOTP Verification failed. '
-            'Continuing to other methods if available', fg='red'
+            "TOTP Verification failed. " "Continuing to other methods if available",
```

Comment:
Changes like these should be merged into one string if they're moved to a single.

Reply:
@phillipjf fixed